deps/wazuh_testing/wazuh_testing/wazuh_variables.py

```diff
@@ -1,20 +0,0 @@
-# Copyright (C) 2015-2021, Wazuh Inc.
```

Comment:
Did you check that it does not affect any different test than fim tests?

Reply:
Yes. All affected tests where refactored. It only affected FIM.